modules/openapi-generator/src/main/resources/go/client.mustache

```diff
@@ -447,6 +447,13 @@ func reportError(format string, a ...interface{}) error {
 	return fmt.Errorf(format, a...)
 }
 
+// Prevent trying to import "bytes"
```

Comment:
Not sure what you mean by this. Any issues using `bytes`?

Reply:
Right, this is a bit confusing. The problem is that in another file `bytes` might or might not be needed as an import. By defining the function here, as bytes is always required then, we avoid unused import / undefined import. This is the same as `fmt.Errorf` but the comment can probably be improved:

```suggestion
// A wrapper for strict JSON decoding
```